frontend/public/components/marketplace/modal-overlay.jsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import {Button} from 'patternfly-react/dist/esm/components/Button';
+import {Modal} from 'patternfly-react/dist/esm/components/Modal';
+import {CatalogItemHeader} from 'patternfly-react-extensions/dist/esm/components/CatalogItemHeader';
+import {PropertiesSidePanel, PropertyItem} from 'patternfly-react-extensions/dist/esm/components/PropertiesSidePanel';
+
+const MarketplaceModalOverlay = (props) => {
```

Comment:
I think a better name might be `MarketplaceItemModal`. Filename should match as well, ie `marketplace-item-overlay.jsx`

Reply:
I agree. Will also re-comment out the marketplace nav item